### DISCUSSION TITLE
`public/sw.js.map` keeps changing after every change in code base.

### DISCUSSION THREAD

**Discussion by HamedSiaban** [DuCanhGH/next-pwa#105] [Q&A]
It's very anoying to commit and push this file changes in every commit.
Should I put this file in `.gitignore`?

**Comment by DuCanhGH** [comment]
@HamedSiaban 
I mean, yeah, they should be .gitignored. You should add this to your `.gitignore`:
```
# next-pwa
**/public/workbox**
**/public/sw**
**/public/worker**
**/public/fallback**
**/public/precache**
```

**Reply by HamedSiaban** [reply]
Thanks man
